frontend/base/src/main/java/com/virtuslab/gitmachete/frontend/icons/MacheteIcons.java

```diff
@@ -0,0 +1,50 @@
+package com.virtuslab.gitmachete.frontend.icons;
+
+import javax.swing.Icon;
+
+import com.intellij.openapi.util.IconLoader;
+
+/**
+ * Icons must be in package named "icons" based on
+ * https://www.jetbrains.org/intellij/sdk/docs/reference_guide/work_with_icons_and_images.html
+ */
+public final class MacheteIcons {
+  private MacheteIcons() {}
+
+  public static final Icon DISCOVER = loadIcon("/icons/macheteLogoIcon.svg");
```

Comment:
While you're here: all paths have `/icons/` prefix and `.svg` extension... maybe just replace `path` param in `loadIcon` with `basename` and add prefix/suffix automatically